service/vpn_key_rotation_webhook_validation_test.go

```diff
@@ -0,0 +1,137 @@
+package service
+
+import (
+	"context"
```

Comment:
Please add ITs for the create and delete webhooks which should check for the events too

Reply:
Where are ITs?